src/store/factory/client.js

```diff
@@ -95,11 +115,21 @@ function createRskClient (asset, network, mnemonic) {
   return createEthereumClient(asset, rskNetwork, rpcApi, scraperApi, EthereumRpcFeeProvider, mnemonic)
 }
 
+function createBSCClient (asset, network, mnemonic) {
+  const isTestnet = network === 'testnet'
+  const bnbNetwork = AssetNetworks.BNB[network]
+  const rpcApi = isTestnet ? 'https://data-seed-prebsc-1-s1.binance.org:8545' : 'https://bsc-dataseed.binance.org'
+  const scraperApi = isTestnet ? 'https://liquality.io/rsk-testnet-api' : 'https://liquality.io/rsk-mainnet-api'
```

Comment:
set this properly, we can set it up later if needed
e.g. `bsc-testnet-api`

Reply:
got it, done